docs/content/reference/ransomware.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Ransomware"
+date: 2021-06-23T18:13:59+05:30
+draft: true
+pre: '<i class="fas fa-lock"></i> '
+weight: 10
+---
+
+The Infection Monkey has the capability of simulating a ransomware attack on your network.
+All actions performed by the encryption routine are safe for production environments.
+
+To ensure minimum intereference and easy recoverability, the ransomware simulation will only run if
+it is configured properly. To do so, you must specify the path to a directory in the configuration.
+If no directory is specified, the simulation will not run.
+
+<!-- add config screenshot here -->
+
+## Which files are encrypted?
+
+All regular files with [relevant extensions](#relevant-file-extensions-for-encryption) in the
+configured directory are attempted to be encrypted during the simulation.
+
+The simulation is not recursive, i.e. it will not touch any files in sub-directories of the
+configured directory — only appropriate files in the top level of the tree.
+
+Symlinks and shortcuts are ignored.
+
+
+## How are the files encrypted?
+
+Files are "encrypted" in place with a simple bit flip. Encrypted files are renamed to have
+`.m0nk3y` appended to their names.
```

Comment:
The `.m0nk3y` extension is the default one right? Shouldn't we also say that that can be changed ?

Reply:
We haven't added the capability to choose your own yet. It might not make it into this release.

Reply:
It's best to update documentation when (or shortly after) functionality changes are made instead of before.